### ISSUE TITLE
💻Add dark mode

### ISSUE THREAD

**Issue by nidhi752** [hedyorg/hedy#5856]
**Idea incl level**
For kids, dark mode can be beneficial in several ways. It reduces eye strain, especially when they're using screens for long periods, such as during online learning or gaming. Lowering blue light exposure can also help protect their sleep patterns, which is crucial for their development and well-being. Additionally, dark mode can make the screen less harsh on their eyes in low-light environments, creating a more comfortable viewing experience. By offering a more visually appealing and less distracting interface, it may also help kids focus better on educational content.

**Comment by Mr-Mysterious001** [comment]
Hi there, can I try to work out this issue?

**Comment by ZealinShubh** [comment]
I can try to do this

**Comment by Felienne** [comment]
Yes, feel free to pick this up @ZealinShubh or @Mr-Mysterious001!

@TiBiBa once tried his hand at this, see: https://github.com/hedyorg/hedy/pull/5105

Maybe that is a useful starting point?

**Comment by nidhi752** [comment]
@Felienne Kindly assign this to me so that i can start working on this.

**Comment by Felienne** [comment]
> @Felienne Kindly assign this to me so that i can start working on this.

Absolutely!

Are you working on this as part of Hacktober or another project?

**Comment by Felienne** [comment]
Do take a look at our docs in the Wiki: https://github.com/hedyorg/hedy/wiki, and join the Discord if you have any more questions (or drop into an online meeting every Friday 3:30pm CET, 9:30 ET)

**Comment by Mr-Mysterious001** [comment]
> Yes, feel free to pick this up @ZealinShubh or @Mr-Mysterious001!
> 
> @TiBiBa once tried his hand at this, see: https://github.com/hedyorg/hedy/pull/5105
> 
> Maybe that is a useful starting point?

Yeah I will have a look at it, thanks btw

**Comment by nidhi752** [comment]
> > @Felienne Kindly assign this to me so that i can start working on this.
> 
> Absolutely!
> 
> Are you working on this as part of Hacktober or another project?

Yes, as part of hackoctoberfest
